src/utils/notifications/slack.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+import { WebhookData } from '../../interfaces/data'
+
+export const sendSlack = async (data: WebhookData) => {
+  try {
+    if (!data.url) throw new Error(`Slack url is not provided`)
+
+    const res = await axios({
+      method: 'POST',
+      url: data.url,
+      data: {
+        text: `*${data.body.alert}*\n\n*URL*: ${data.body.url}\n*TIME*: ${data.body.url}\n`,
```

Comment:
should be `data.body.time` right?

```suggestion
        text: `*${data.body.alert}*\n\n*URL*: ${data.body.url}\n*TIME*: ${data.body.time}\n`,
```

Reply:
Ah right! Thanks!